runtime/codert_vm/cnathelp.cpp

```diff
@@ -2517,6 +2517,18 @@ fast_jitInstanceOf(J9VMThread *currentThread, J9Class *castClass, j9object_t obj
 	return isInstance;
 }
 
+UDATA J9FASTCALL
+#if defined(J9VM_ARCH_X86) || defined(J9VM_ARCH_S390)
+/* TODO Will be cleaned once all platforms adopt the correct parameter order */
+fast_jitCheckAssignable(J9VMThread *currentThread, J9Class *clazz, J9Class *castClass)
+#else /* J9VM_ARCH_X86 || J9VM_ARCH_S390*/
+fast_jitCheckAssignable(J9VMThread *currentThread, J9Class *castClass, J9Class *clazz)
+#endif /* J9VM_ARCH_X86 || J9VM_ARCH_S390*/
```

Comment:
Are there some braces missing around this area? Not really sure what is going on but looking at the surrounding code this appears to be incorrect at first glance. Perhaps I'm missing something.

Reply:
I don't think extra braces are need here. It looks very similar to the method above `fast_jitInstanceOf` and compiles.